packages/devtools-reps/src/reps/grip-map.js

```diff
@@ -105,14 +129,10 @@ function entriesIterator(props, object, max) {
   let entries = getEntries(props, mapEntries, indexes);
   if (entries.length < getLength(object)) {
     // There are some undisplayed entries. Then display "…".
-    entries.push(span({
-      key: "more",
-      className: "more-ellipsis",
-      title: "more…"
-    }, "…"));
+    entries.push(getMoreEllipsisElement());
   }
 
-  return unfoldEntries(entries);
+  return unfoldEntries(entries, props.mode);
```

Comment:
we make no use of the mode in `unfoldEntries`. Do I miss something ?

Reply:
at some point i tried to calculate sth in there. Will remove.